### PR TITLE
Add environment files for windows and linux

### PR DESCRIPTION
**Pull request by lauramurgatroyd**

### CODE REVIEW

File: environments/idvc22.3.0.linux.yml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  - yaml=0.2.5=h516909a_0
+  - zipp=3.6.0=pyhd8ed1ab_0
+  - zlib=1.2.11=h36c2ea0_1013
+  - zstd=1.3.3=1
+  - pip:
+    - cil-data==22.0.0
+    - pyqt5-sip==4.19.18
+    - pyqtchart==5.12
+    - pyqtwebengine==5.12.1
+prefix: /home/m1116300/.conda/envs/base-env/envs/idvc
```
Comment (paskino) [inline environments/idvc22.3.0.linux.yml:127]: these are suspicious

Comment (lauramurgatroyd) [reply environments/idvc22.3.0.linux.yml:127]: https://stackoverflow.com/questions/41274007/anaconda-export-environment-file

Seems like it's fine to leave but I will remove and test again

File: environments/idvc22.3.0.windows.yaml [hunk 2]
```diff
@@ -0,0 +1,124 @@
+name: idvc_windows2
+channels:
+  - paskino
```
Comment (paskino) [inline environments/idvc22.3.0.windows.yaml:3]: swap `paskino` and `ccpi` channels

File: environments/idvc22.3.0.windows.yaml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  - ccpi
```
Comment (lauramurgatroyd) [inline environments/idvc22.3.0.windows.yaml:4]:
```suggestion
  - ccpi
  - paskino
```

File: environments/idvc22.3.0.linux.yml [hunk 4]
```diff
@@ -0,0 +1,126 @@
+name: idvc
+channels:
+  - paskino
+  - ccpi
```
Comment (lauramurgatroyd) [inline environments/idvc22.3.0.linux.yml:4]:
```suggestion
  - ccpi
  - paskino
```

**Pull request merged**